features/Auth/userSlice.ts

```diff
@@ -3,17 +3,17 @@ import { createSlice, PayloadAction } from '@reduxjs/toolkit';
 export interface userState {
     token: string;
     isLoggedIn: boolean;
-    name: string;
-    phone: string;
-    email: string;
+    // name: string;
```

Comment:
remove these comments

Reply:
sure

Reply:
i have kept them as in future we might need them so only uncomment it 😅